README.md

```diff
@@ -48,15 +48,28 @@ C:\T\Pictures\IMG_5678.HEIC -> IMG_5678.HEIC.jpg
 ### Renaming Files
 
 Sometimes files have the wrong extension.
-To rename JPEG-encoded files that don't have the standard `.jpg` extension, use the `-FixExtensionIfJpeg` switch.
-(The `=>` in the output indicates that the file was renamed vs. converted.)
+To rename JPEG-encoded files that don't have the standard `.jpg` extension, use the `-FixExtensionIfJpeg` switch (alias `-f`).
+
+The `=>` in the output indicates that the file was renamed vs. converted.
 
 ```PowerShell
 PS C:\T> dir C:\T\Pictures\*.HEIC | .\ConvertTo-Jpeg.ps1 -FixExtensionIfJpeg
 C:\T\Pictures\IMG_1234 (Edited).HEIC => IMG_1234 (Edited).jpg
 C:\T\Pictures\IMG_1234.HEIC -> IMG_1234.HEIC.jpg
 ```
 
+### Removing existing extensions
+
+If you want to remove the existing extension of a file, use the `-RemoveExistingExtension` switch (alias `-r`).
+
+This will not touch the original file, it will only change the output.
```

Comment:
I don't think this line is needed as the README already says it doesn't modify the original?